process_request/tests.py

```diff
@@ -55,6 +67,39 @@ def test_machineuser(self):
         self.assertEqual(str(user), system)
 
 
+class TestHelpers(TestCase):
+
+    def test_get_container_indicators(self):
+        letters = ''.join(random.sample(string.ascii_lowercase, random.randint(2, 10)))
+        title = "Digital Object: " + letters
+        instance = {'instance_type': 'digital_object', 'digital_object': {'_resolved': {'title': letters}}}
+        indicator = get_container_indicators(instance)
+        self.assertEqual(indicator, title)
+
+        type = ''.join(random.sample(string.ascii_lowercase, random.randint(2, 10))).capitalize()
```

Comment:
The problem here is that you're not testing the `capitalize()` in `get_container_indicators()`.  I'd call `capitalize()` below in line 82 instead.